app/views/users/show.html.erb

```diff
@@ -10,26 +10,29 @@
   <section>
     <h1>Bookmarked Segments</h1>
   </section>
-</section>
-
 <section id="github_section">
   <h2>Github:</h2>
-  <h4>Repositories:</h4>
-  <% @github_info.get_repositories.each do |repo| %>
-    <section id="repo">
-     <p><%= link_to repo.name, repo.html_url %></p>
-   </section>
-  <% end %>
-  <h4>Followers:</h4>
-  <% @github_info.get_followers.each do |follower| %>
-    <section id="followers">
-      <p><%= link_to follower.login, follower.html_url %></p>
-    </section>
-  <% end %>
-  <h4>Following:</h4>
-  <% @github_info.get_following.each do |following| %>
-    <section id="following">
-      <p><%= link_to following.login, following.html_url %></p>
-    </section>
-  <% end %>
+  <% if current_user.token.nil? %>
```

Comment:
Added this conditional to the view so that a user who is not logged in to github with a token will not be able to see any github information and will be informed that they will need to sign in in order to see that information.